Extensions/xkit_patches.js

```diff
@@ -1670,7 +1670,8 @@ XKit.tools.getParameterByName = function(name){
 					XKit.tools.add_css("." + class_name + ":after {" +
 							" background-image: url('" + icon + "') !important;" +
 							" background-size: auto auto !important;" +
-							" margin-top: -9px !important; " +
+							" margin-top: -7px !important; " +
+							" opacity: 1 !important; " +
```

Comment:
I've modified every line with an opacity value, but  the only way I could get it to work is by adding that line in the js file. I'll keep working on it!